src/DevChatter.DevStreams.Infra.Dapper/Services/ChannelAggregateService.cs

```diff
@@ -107,5 +107,23 @@ public async Task<int> Delete(int id)
                 return await connection.DeleteAsync<Channel>(id);
             }
         }
+
+        public async Task<string> LuckySearch()
+        {
+            /*const string sql =
+                @"select c.Name from Channels c
+                  inner join ScheduledStreams ss on ss.ChannelId = c.Id
+                  where CURRENT_TIMESTAMP between ss.LocalStartTime and ss.LocalEndTime
+                  ORDER BY RAND() LIMIT 1";*/
+
+            const string sql = @"select TOP 1 Name from Channels ORDER BY RAND()";
```

Comment:
Good way to handle this (sans the obvious not accounting for live that you mentioned).